dtests/src/resources/regression_docs/HowToRunSnappyHydraTests.md

```diff
@@ -0,0 +1,84 @@
+# How To Run Snappy Hydra Tests
+
+## Running battery tests
+
+#### Pre-requisites:
+
+ 1. Build the product using ./gradlew clean buildAll
+
+ 2. Modify the local.conf files as per requirement and build the dtests using ./gradlew buildDtests.
+
+ 3. Make sure that no java processes are running on the machines
+
+ 4. Result directory path needs to be already created for storing the logs of test run
+
+**Note:** SnappyData is build on 64-bit JVM and tests are also run using 64-bit JVM
+
+#### Execute below command to run any snappy hydra bt
+
+```
+export SNAPPYDATA_SOURCE_DIR=<product checkout dir>
+$SNAPPYDATA_SOURCE_DIR/sample-runbt.sh <result-directory-path> $SNAPPYDATA_SOURCE_DIR [-l <local-conf-file-path> -r <num-times-to-run-test> -m <mailAddresses>] <list-of-bts>
```

Comment:
Here too ,If -m (mailAddresses) option does not work,then it is better to remove it .